pubpub/pubpub
# dev: Remove keen for good and update admin dashboard

This removes all remaining Keen call-sites and references, including:

- hydrateWrapper
- cookie settings (text and actual cookie name)
- admin dashboard

The admin dashboard stopped functioning when we removed Keen, so I updated it to use the new Metabase dashboard.

_Test plan_
- Make sure there are no Keen calls on any pageload
- Make sure you can toggle cookie preferences, and when doing so Heap cookies still appear/disappear, and the cookie opt-out/in we drop is still set properly
- Visit `/admin` from a community (e.g. demo) and make sure it 404s
- Visit `/admin` on the www community while logged out and make sure it 404s
- Visit `/admin` on the www community while logged into your personal account or the pubpub team account and make sure the dashboard loads
- Visit `/admin` on the www community while logged into a test account that is not your personal or the pubpub team account and make sure it 404s
